cli/src/graphql/generated/queries.ts

```diff
@@ -66,6 +66,7 @@ export type DataObjectInfoFragment = {
         video?: Types.Maybe<{ id: string }>
         subtitle?: Types.Maybe<{ id: string }>
       }
+    | { __typename: 'DataObjectTypeChannelPayoutsPayload' }
```

Comment:
This type name does not exist in `master` branch yet, this is part of [ephesus](https://github.com/Joystream/joystream/blob/5b86f0cb97606d5056af096eab5a0ae6f16a4ff0/query-node/schemas/storage.graphql#L176-L177) QN. Seems like `yarn generate:all` was run while being in the ephesus branch. 
Please verify it, also make sure to run `yarn build:packages` in the monorepo root, to ensure that old build artifacts aren't used in generating gql schema.ts/queries.ts files

Reply:
Inded it seems I didn't run build:packages after switching branches, before running yarn generate:all, pushed update which now seems to only update Float -> BigInt types in the schema.ts files